src/build/__main__.py

```diff
@@ -364,10 +359,16 @@ def main(cli_args: Sequence[str], prog: Optional[str] = None) -> None:  # noqa:
             built = build_call(
                 args.srcdir, outdir, distributions, config_settings, not args.no_isolation, args.skip_dependency_check
             )
-            artifact_list = _natural_language_list([f'[underline]{artifact}[reset][bold][green]' for artifact in built])
-            _print(f'[bold][green]Successfully built {artifact_list}')
+            artifact_list = _natural_language_list(
+                ['{underline}{}{reset}{bold}{green}'.format(artifact, **_STYLES) for artifact in built]
+            )
+            print('{bold}{green}Successfully built {}{reset}'.format(artifact_list, **_STYLES))
     except Exception as e:  # pragma: no cover
-        _print('\n[dim]{}[reset]\n'.format(traceback.format_exc().strip('\n')))
+        colored_tb = '\n{dim}{}{reset}\n'.format(
+            traceback.format_exc().strip('\n'),
+            **_STYLES,
+        )
+        print(colored_tb)
```

Comment:
Personally, I'd make black happy this way:

```suggestion
        tb_str = traceback.format_exc().strip('\n')
        print('\n{dim}{}{reset}\n'.format(tb_str, **_STYLES))
```